controller/iteration_blackbox_test.go

```diff
@@ -348,13 +347,19 @@ func (rest *TestIterationREST) createWorkItem(parentSpace space.Space) workitem.
 
 func (rest *TestIterationREST) TestShowIterationModifiedUsingIfModifiedSinceHeaderAfterWorkItemLinking() {
 	// given
-	fxt := createSpaceAndRootAreaAndIterations(rest.T(), rest.DB)
+	fxt := tf.NewTestFixture(rest.T(), rest.DB,
+		append(createSpaceAndRootAreaAndIterations(),
+			tf.WorkItems(1,
+				tf.SetWorkItemField(workitem.SystemState, workitem.SystemStateNew),
```

Comment:
Why do we specifically need to set `workitem.SystemState` here? Its not used anywhere in the test case further.

Reply:
Good catch. It is not needed and I've removed it here: b66518a1.